### DISCUSSION TITLE
alwatr-cli

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#72] [Ideas]
من هنوز قانع نشدم که آیا واقعا cli لازم هستش یا نه
ولی به نظرم یک generator که برای فرد یک کلاس element جدید بسازه یا مثلا برای من pakcage خام بسازه جذابه
و میشه از این پروژه اساسی ایده گرفت و یا حتی با خودش ساخت
لطفا نظر بدید و اگر خواستید ادامه بدید

https://open-wc.org/docs/development/generator/

**Comment by alimd** [comment]
سورس generator اشون دقیقا کجاست؟

**Reply by MM25Zamanian** [reply]
نمیدونم میگردم ولی فکر میکنم باید تو دل cli باشه؟

**Reply by MM25Zamanian** [reply]
[Polymer cli](https://github.com/Polymer/polymer-cli)

**Comment by alimd** [comment]
اینو برای شما گذاشتم لطفا ادامه بده
@MM25Zamanian

**Reply by MM25Zamanian** [reply]
انشالله یه سورس خوب واسه cli پیدا کنم شروع میکنم

**Reply by MM25Zamanian** [reply]
This [Generator](https://github.com/code-workshop-kit/create) is nice??

**Reply by alimd** [reply]
بابا همون لینک که دادم کامل توضیح داده دیگه
دقیق بخون

**Comment by alimd** [comment]
another inpiration sample https://github.com/Simplr/Configs/tree/main/create

**Comment by alimd** [comment]
Wow
https://github.com/lit/lit/pull/2647
